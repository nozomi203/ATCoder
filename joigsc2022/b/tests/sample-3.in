10
1 G
2 J
3 G
4 O
7 G
9 J
10 G
14 I
17 G
19 G
7
11 6
6 3
17 19
1 18
17 17
20 1
20 10
