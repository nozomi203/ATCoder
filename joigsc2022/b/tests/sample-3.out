25
27
28
17
26
39
30
